packages/Demo/Entity/RectorRun.php

```diff
@@ -18,7 +18,7 @@
 use Rector\Website\Exception\ShouldNotHappenException;
 use Symfony\Bridge\Doctrine\IdGenerator\UuidGenerator;
 use Symfony\Component\Uid\Uuid;
-use Symplify\SymplifyKernel\Strings\StringsConverter;
+use Symplify\PackageBuilder\Strings\StringFormatConverter;
```

Comment:
@TomasVotruba I updated to use the `PackageBuilder` StringFormatConverter as `StringsConverter` seems functionality is in there and `SymplifyKernel'`s StringsConverter no longer exists in version `^10.0`

Reply:
Yes, thank you. That was intended only for internal Kernel naming, not sure what is doing here.